ros_cross_compile/docker/build_workspace.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -euxo pipefail
+
+source /opt/ros/${ROS_DISTRO}/setup.bash
+colcon build --mixin ${TARGET_ARCH}-docker \
+  --build-base build_${TARGET_ARCH} \
+  --install-base install_${TARGET_ARCH}
+chown -R ${OWNER_USER} .
```

Comment:
shellcheck / add missing quotes around the bash variables. If one day we forget to set one of those variables, we'll get much better error reporting.